specification/securitycenter/resource-manager/readme.md

```diff
@@ -0,0 +1,60 @@
+# Security Center
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Security.
+
+
+
+---
+## Getting Started 
+To build the SDK for Security, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the Security API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-08-01-preview
+```
+
+
+### Tag: package-2017-08-01
+
+These settings apply only when `--tag=package-2017-08-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-08-01-preview'
+input-file:
+- Microsoft.Security/preview/2017-08-01-preview/security.json

```

Comment:
Please add 2015 version as well so SDK will be generated with both APIs. For example https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/readme.md#composite-packages